woof-code/rootfs-petbuilds/geany/petbuild

```diff
@@ -1,13 +1,13 @@
 download() {
-    [ -f geany-1.38.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-1.38.tar.bz2
+    [ -f geany-2.0.tar.bz2 ] || wget -t 3 -T 60 https://download.geany.org/geany-2.0.tar.bz2
 }
 
 build() {
-    tar -xjf geany-1.38.tar.bz2
-    cd geany-1.38
+    tar -xjf geany-2.0.tar.bz2
+    cd geany-2.0
     patch -p1 < ../defaults.patch
-    ./configure --prefix=/usr --disable-plugins --disable-vte
+    ./configure --prefix=/usr --enable-plugins --enable-vte
```

Comment:
The plugins are big and make Geany slower to start - traditionally, Puppy doesn't have them

Reply:
Ok, so that's why I wasn't seeing any increase in ISO size when enabling plugins. What about VTE?

Reply:
I believe most people who use Geany on Puppy don't use it as an IDE but as a text editor, and doubt the usefulness of the intergrated terminal